fixtures/backup/model_dependencies/detailed.json

```diff
@@ -6218,6 +6218,34 @@
     "table_name": "workflow_engine_datasource",
     "uniques": []
   },
+  "workflow_engine.datasourcedetector": {
+    "dangling": false,
+    "foreign_keys": {
+      "data_source": {
+        "kind": "FlexibleForeignKey",
+        "model": "workflow_engine.datasource",
+        "nullable": false
+      },
+      "detector": {
+        "kind": "FlexibleForeignKey",
+        "model": "workflow_engine.detector",
+        "nullable": false
+      }
+    },
+    "model": "workflow_engine.datasourcedetector",
```

Comment:
do we have any patterns or nomenclature around lookup tables / is that something we should start doing? (i've seen stuff like `datasource2detector`, `datasource_detector`, and the verbose `datasource_to_detector`). not strong thoughts / opinions on this, just curious.

Reply:
The usual pattern in the code base is to just mash the model names together. So `ProjectTeam` for example. https://github.com/getsentry/sentry/blob/2bfa30d041babf147c8fa28f68b737bbf13e766b/src/sentry/models/project.py#L238